examples/notebooks/Date Picker Widget.ipynb

```diff
@@ -817,21 +807,21 @@
    ]
   ],
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
```

Comment:
Luckily, I don't think you have to reopen and re-save every one.  Instead, you can probably get away with altering this by hand.